### ISSUE TITLE
Regress in 1.74: Cannot open submodule, nothing happened

### ISSUE THREAD

**Issue by Ben-Mack** [ForkIssues/TrackerWin#1542]
Nothing happened when right click and select Open 'submodule name' after updating to GitFork 1.74

(Reinstalling does not help but also lose all of my previous config, account... How can I recover these?)

**Comment by Ben-Mack** [comment]
After upgrading, there's a dialog saying about unsafe repo and I have to click mark as safe to continue

**Comment by Ben-Mack** [comment]
Log:
```
? 2022-05-22 15:02:57.056 Cannot parse reference refs/replace/fea60b579994c0eb9d2256b40e7c040bd3a18ce3
? 2022-05-22 15:02:57.056 Cannot parse reference refs/replace/fea6efd0900e9321c0f232a00de9e3f761f28fd4
? 2022-05-22 15:02:57.056 Cannot parse reference refs/replace/fecb709ad482766126f6f9f5a6ec9d87f04db0da
? 2022-05-22 15:02:57.056 Cannot parse reference refs/replace/ff4030f089fadcca6ad811540f1dfc9b87d02f11
```

**Comment by Ben-Mack** [comment]
Could you please provide installation exe of previous version? I can't work right now because no submodule could be opened

**Comment by DanPristupov** [comment]
Try to open the submodule repo using the Ctrl+O dialog. I'm working on a hotfix.

**Comment by Ben-Mack** [comment]
Thank you! Ctrl+O workaround worked. The unsafe repo dialog appears when opening each submodule, after marking as safe then I can open these submodule normally now

**Comment by DanPristupov** [comment]
I've just released the 1.74.1 hotfix.

**Comment by Ben-Mack** [comment]
I've updated and confirm this is fixed. Thank you for such a quick fix!

You might want to take a look at preserving user config when reinstalling.
Here my flow:
I didn't uninstall the current Fork.
I manually download and run the exe provided in the homepage. 
I thought there'll be a dialog saying keep user config or so but it's just auto installing then display a dialog to enter name/email. All of my previous config is cleared then.

Also, it'd be great if Fork backup config weekly to prevent similar situation.
